chapter_natural-language-processing/sentiment-analysis-cnn.md

```diff
@@ -0,0 +1,299 @@
+# 文本分类：情感分析
+
+文本分类即把一段不定长的文本序列变换为类别。在文本分类问题中，情感分析是一项重要的自然语言处理任务。例如，Netflix或者IMDb可以对每部电影的评论进行情感分类，从而帮助各个平台改进产品，提升用户体验。

```

Comment:
This paragraph is the same as the previous one. Can you just give a brief intro and highlight the difference?